### PR TITLE
#534:open popover on hover also shown in menuhover in storybook

### PR DESCRIPTION
**Pull request by jtanya34**
The aim of this PR is to add an hover functionality on popover. The example is added as menuHover in menu stories.

### CODE REVIEW

File: src/popover/src/Popover.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export default class Popover extends Component {
      * When true, the Popover is manually shown.
      */
     isShown: PropTypes.bool,
+    isHover: PropTypes.bool,
```
Comment (mshwery) [inline src/popover/src/Popover.js]: Can you add some documentation to the prop, and maybe we could rename to `triggerOnHover`?

**Comment by jtanya34** [comment]
> What would you think about providing this via an enum prop called `trigger`? That way we could easily distinguish between opening based on click or hover (or other mechanisms in the future)?
> 
> `trigger: 'click' | 'hover'` where `'click'` is the default value?

Good Idea, I made the changes accordingly.

File: src/popover/src/Popover.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ export default class Popover extends Component {
      * When true, the Popover is manually shown.
      */
     isShown: PropTypes.bool,
+    trigger: PropTypes.oneOf(['click', 'hover']),
```
Comment (mshwery) [inline src/popover/src/Popover.js]: Could you add a jsdoc description for the prop here?

Comment (jtanya34) [reply src/popover/src/Popover.js]: done!

Comment (mshwery) [reply src/popover/src/Popover.js]: Oh, that reminds me – maybe we should add `trigger: 'click'` to `defaultProps`? I realize with the current implementation it's not strictly necessary, but would be good for semantics I think.

Comment (jtanya34) [reply src/popover/src/Popover.js]: done

**Comment by robphoenix** [comment]
🎉

**Pull request merged**